### ISSUE TITLE
Add a banner to the top of select pages within the self service wizard

### ISSUE THREAD

**Issue by ThomasKranitsas** [topcoder-platform/micro-frontends-self-service-app#190]
### Requirement: 

- Business wants to add a ‘website design' banner to select pages within the self service wizard. These pages are:
  - Basic info step
  - Review step
- At this time, we do NOT want to apply this to all pages, only these two. However, in the future, the business may want to easily add the banner to other steps/pages in the wizard. Having an easy configuration in a database table of some sort, which can be flipped easily from true/false, per page, is a nice to have requirement. 

Content (Note -> The title and description text are the same between the to pages)
- Title Text: WEBSITE DESIGN (all capital letters)
- Description Text:  Create a beautiful custom visual design for your website. Specify the scope and device types, your vision, and receive up to 5 modern designs.

### User Interface Requirements:

- Basic Info page in wizard:
![image-20220218-181841](https://user-images.githubusercontent.com/4612921/155341988-68958859-55e3-4361-ba36-6b80fdaee8bc.png)

- Review page in wizard:
![image-20220218-181910](https://user-images.githubusercontent.com/4612921/155341999-c09d7102-58fb-4bed-b76f-b7766be54e08.png)

The complete Figma file with the design is here:  https://www.figma.com/file/61RVR37D1ifWPeeNdLH393/Work-App---MVP-Handoff-(Ray's-File)?node-id=2338%3A6400

Context: The UX team’s strong recommendation is that we link prospective customers directly to step 1 of the website design wizard from any advertisements. Adding additional information and context of where the customer is once they land on step 1 from clicking an ad is important both from an experience and Google algorithm perspectives.
